torch/onnx/utils.py

```diff
@@ -807,45 +819,45 @@ def _run_symbolic_function(g, n, inputs, env, operator_export_type=OperatorExpor
                 domain = 'caffe2'
+            symbolic_fn = _find_symbolic_fn(ns, domain, op_name, opset_version, operator_export_type)
+            if symbolic_fn is None:
+                return symbolic_fn
             attrs = {k: n[k] for k in n.attributeNames()}
-
-            if not sym_registry.is_registered_op(op_name, domain, opset_version):
-                warnings.warn("ONNX export failed on quantized operator {}::{} because "
-                              "torch.onnx.symbolic_opset{}.{} does not exist. "
-                              .format(ns, op_name, opset_version, op_name))
-            op_fn = sym_registry.get_registered_op(op_name, domain, opset_version)
-            return op_fn(g, *inputs, **attrs)
+            return symbolic_fn(g, *inputs, **attrs)
 
         # custom ops
         elif sym_registry.is_registered_version(ns, opset_version):
-            if not sym_registry.is_registered_op(op_name, ns, opset_version):
-                warnings.warn("ONNX export failed on custom operator {}::{} because "
-                              "torch.onnx.symbolic_opset{}.{} does not exist. "
-                              "Have you registered your symbolic function with "
-                              "torch.onnx.register_custom_op_symbolic(symbolic_name, symbolic_fn)?"
-                              .format(ns, op_name, opset_version, op_name))
-            symbolic_fn = sym_registry.get_registered_op(op_name, ns, opset_version)
+            domain = ns
+            error_msg = "Have you registered your symbolic function with " \
+                        "torch.onnx.register_custom_op_symbolic(symbolic_name, symbolic_fn)?"
+            symbolic_fn = _find_symbolic_fn(ns, domain, op_name, opset_version, operator_export_type,
+                                            error_msg)
+            if symbolic_fn is None:
+                return symbolic_fn
             attrs = {k: n[k] for k in n.attributeNames()}
             return symbolic_fn(g, *inputs, **attrs)
-
         else:
-            warnings.warn("ONNX export failed on an operator with unrecognized namespace {}::{}; "
-                          "If you are trying to export a custom operator, make sure you registered "
-                          "it with the right domain and version."
-                          "Otherwise please report a bug".format(ns, op_name))
+            if operator_export_type != OperatorExportTypes.ONNX_ATEN_FALLTHROUGH:
+                warnings.warn("ONNX export failed on an operator with unrecognized namespace {}::{}; "
+                              "If you are trying to export a custom operator, make sure you registered "
+                              "it with the right domain and version."
+                              "Otherwise please report a bug".format(ns, op_name))
             return None
-
+    except RuntimeError:
+        if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLTHROUGH:
```

Comment:
All runtime errors are hidden for `ONNX_ATEN_FALLTHROUGH`?

Reply:
All runtime errors that are coming from symbolics are hidden. We will export the op as is, even if the spec is not known to the exporter.